### ISSUE TITLE
Accessibility support on Linux

### ISSUE THREAD

**Issue by ialokim** [JetBrains/compose-multiplatform#2167]
As mentioned in https://github.com/JetBrains/compose-jb/issues/1366#issuecomment-992295089, accessibility support on Linux seems to be difficult. Right now, screen readers like Orca completely ignore windows created using Compose for Desktop. I understand, e.g. from [here](https://wiki.debian.org/accessibility#Java_accessibility) and [here](https://wiki.gnome.org/Attic/Java%20Access%20Bridge), that support for accessibility services should be possible under systems like Debian and Ubuntu (with some manual tweaks).

What exactly is especially difficult on Linux? Having at least basic accessibility support there as well would be great!

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2167">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
